packages/fuels/tests/configurables.rs

```diff
@@ -0,0 +1,170 @@
+use fuels::prelude::*;
+use fuels_types::SizedAsciiString;
+
+#[tokio::test]
+async fn contract_uses_default_configurables() -> Result<()> {
+    abigen!(Contract(
```

Comment:
If you want, you can use setup contract test macro here.

Reply:
I would rather keep the same structure because I show that if I do not modify anything the contract will use the default values. I mean, the setup macro would do the same but still I want it to be visible.